authentication/index.adoc

```diff
@@ -0,0 +1,65 @@
++
+ ** An `Authorization: Bearer…​` header.
+ ** A WebSocket subprotocol header for the web socket requests.
+ ** The built-in OAuth server through specific endpoints.
++
+Users can also xref:../authentication/managing-oauth-access-tokens.adoc#managing-oauth-access-tokens[view and manage their own OAuth acces tokens].
+
+* An X.509 client certificate - You require an HTTPS connection to the API server to use the X.509 client certificates for authentication.
+
+[NOTE]
+====
+If you do not present a valid access token or certificate, your request is unauthenticated and you receive an HTTP 401 error.
+====
+An administrator can configure authentication through the following tasks:
+
+* Configuring an identity provider: You can define any xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[supported identity provider in {product-title}] and add it to your cluster.
+* xref:../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[Configuring the internal OAuth server]: The {product-title} control plane includes a built-in OAuth server that determines the user’s identity from the configured identity provider and creates access tokens. You can configure the token duration and inactivity timeout, and customize the internal OAuth server URL.
+* xref:../authentication/configuring-oauth-clients.adoc#oauth-register-additional-client_configuring-oauth-clients[Registering an OAuth client]: {product-title} includes several xref:../authentication/configuring-oauth-clients.adoc#oauth-default-clients_configuring-oauth-clients[default OAuth clients]. You can register and configure additional OAuth clients.
++
+[NOTE]
+====
+Users must specify either a default or custom OAuth client that receives and uses the token in their requests for the OAuth token.
+====
+
+* Managing cloud provider credentials using the xref:../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc#about-cloud-credential-operator[Cloud Credentials Operator].
+
+[id="authorization-overview"]
+== About authorization in {product-title}
+Authorization involves determining whether a user has the required permissions to perform an action.
+
+Based on how users interact with an {product-title} cluster, there are xref:../authentication/understanding-authentication.adoc#rbac-users_understanding-authentication[three types of users]: regular users, system users, and service accounts. By default, each user type has a set of access permissions and privileges. These permissions and privileges are represented through the xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[role-based access control (RBAC) objects], such as rules, roles, and bindings.
+
+Administrators can manage authorization using the RBAC objects. You can also control access to an {product-title} cluster through xref:../authentication/using-rbac.adoc#rbac-projects-namespaces_using-rbac[projects and namespaces].
```

Comment:
Administrator's can't manage the full extent of authorization only using the RBAC objects. There are SCCs, too, for example, and there might be more mechanisms in the future.

"Administrators can manage user access to cluster resources using the RBAC objects." would perhaps be a more accurate statement?

Reply:
Comment addressed.